SUMMARY.md

```diff
@@ -1,15 +1,16 @@
 # Tessel 2 Documentation
 
 - API
-    * [CLI](/API/CLI.md)
-    * [Hardware API](/API/Hardware_API.md)
-    * [Languages](/API/Languages.md)
-    * [Modules](/API/Modules.md)
+  * [CLI](API/CLI.md)
+  * [Hardware API](API/Hardware_API.md)
+  * [Languages](API/Languages.md)
+  * [Modules](API/Modules.md)
+  * [Network API](API/Network_API.md)
 - Debugging
-    * [LAN Discovery](/Debugging/LAN_Discovery.md)
-    * [Repos](/Debugging/Repos.md)
-    * [USB](/Debugging/USB.md)
-  * [Introduction](/Introduction.md)
+  * [LAN Discovery](debugging/LAN_Discovery.md)
+  * [Repos](debugging/Repos.md)
+  * [USB](debugging/USB.md)
 - Tutorials
-    * [Communication Protocols](/Tutorials/Communication_Protocols.md)
-    * [Making Your Own Module](/Tutorials/Making_Your_Own_Module.md)
+  * [Communication Protocols](Tutorials/Communication_Protocols.md)
+  * [Making Your Own Module](Tutorials/Making_Your_Own_Module.md)
+* [Introduction](Introduction.md)
```

Comment:
Couldn't fix it. The problem is `gitbook-summary` takes the directory structure and `book.json` to generate the `SUMMARY.md` file, which is ordered alphabetically by directories first and then individual files. Then `gitbook` also uses `book.json` and `SUMMARY.md` to generate the static site, and since the `book.json` configuration tells `gitbook` to replace the README with `Introduction.md`, which is already included in the `SUMMARY.md`, then `Introduction.md` becomes the `index.html` of the site but stays last in the navigation (based on the summary file). 

I thought the solution to this would be to tell `gitbook-summary` to ignore `Introduction.md` but the `ignores` configuration only works on directories at the moment. I may open a PR on that plugin to allow file paths to that configuration, so we don't have this problem anymore.